tooling/features/io.siddhi.distribution.editor.core.feature/src/main/resources/docker-export/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+
+# use siddhi-runner-base
+FROM siddhiio/siddhi-runner-base-alpine:5.1.0-m2
```

Comment:
We should not hard-code the base image version. Can we use the ${project.version} here